### PR TITLE
feat: WriteData page for homepage experience

### LINKED ISSUE #3891: firstMile: Wizard Step 5b: Listen for Data
User Story: Step 5: Write Data (Listen Portion of the page)

The user can see whether their data has been written or if the system is still "listening for data..."
    -  When the user clicks the "Next" button, navigate the user to the next step in the wizard and checkmark the overview step as being completed (blue checkmark)
    - When the user clicks the "PREVIOUS" button, navigate the user to the previous step in the wizard

Dev Notes
    - The listening for data component will resemble the telegraf testing configuration overlay

### LINKED ISSUE #3892: firstMile: Wizard Step 6: Execute a Flux Query
User Story: Step 6: Execute a Flux Query

    ACs

    - When the user is on the Execute a Flux Query step, highlight that portion of the wizard
    - Display the text from the Figma Prototype
    - The user can use a copy code component to copy/paste code snippets as written in the Figma prototype
    -  When the user clicks the "Next" button, navigate the user to the next step in the wizard and checkmark the overview step as being completed (blue checkmark)
    - When the user clicks the "PREVIOUS" button, navigate the user to the previous step in the wizard

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/ui/issues/3890
Closes #3891 
Closes #3892 

https://user-images.githubusercontent.com/18511823/157506145-1db8d676-6a11-4bab-8848-c7dfd8bdd330.mov

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=4036)

File: src/homepageExperience/components/DataListening.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+class DataListening extends PureComponent<Props, State> {
+  private intervalID: ReturnType<typeof setInterval>
+  private startTime: number
+  private timer: ReturnType<typeof setInterval>
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      loading: LoadingState.NotStarted,
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/DataListening.tsx:42]: This was mostly copy/paste from the existing file. I just removed the button and triggered the `listen` when a bucket is selected. 

Also re-triggers the check when bucket is changed.

File: src/homepageExperience/components/ConnectionInformation.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      case LoadingState.NotFound:
+        return 'Data Not Found'
+      case LoadingState.Error:
+        return 'Error Listening for Data'
+    }
+  }
+
+  private get additionalText(): any {
+    const docs = (
+      <span> Make sure the correct bucket is selected and then retry. </span>
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/ConnectionInformation.tsx:61]: This file was entirely reused, with only this line changed.

Comment (ChitlangeSahas) [reply src/homepageExperience/components/ConnectionInformation.tsx:61]: (It was referencing to telegraf configuration before, I changed it to a more relevant text for the homepage experience.)

File: src/homepageExperience/components/DataListening.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  bucket: string
+}
+
+interface State {
+  loading: LoadingState
+  timePassedInSeconds: number
+  secondsLeft: number
+  previousBucket: string
+  retry: boolean
+}
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/DataListening.tsx:23]: This file was also reused, with minor changes: 

1. The Listen for data button was removed.
2. A retry button was added, for when the request times out, the user can retry.

**Pull request merged**